docs/docs_skeleton/docs/guides/langsmith/index.md

```diff
@@ -5,8 +5,8 @@ import DocCardList from "@theme/DocCardList";
 LangSmith helps you trace and evaluate your language model applications and intelligent agents to help you
 move from prototype to production.
 
-Check out the [interactive walkthrough](walkthrough) below to get started.
```

Comment:
@baskaryan do you know if the extra should be pulled here so that it ends up being a relative import? Or is the approach with notebooks different?